samples/javascript_nodejs/23.facebook-events/index.js

```diff
@@ -1,92 +1,66 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-const { BotFrameworkAdapter, ConversationState, MemoryStorage, UserState } = require('botbuilder');
-const { BotConfiguration } = require('botframework-config');
+// index.js is used to setup and configure your bot
+
+// Import required packages
 const path = require('path');
 const restify = require('restify');
-const FacebookEventsBot = require('./bot');
-
-// Read botFilePath and botFileSecret from .env file.
-// Note: Ensure you have a .env file and include botFilePath and botFileSecret.
-const ENV_FILE = path.join(__dirname, '.env');
-require('dotenv').config({ path: ENV_FILE });
-
-// Create HTTP server.
-let server = restify.createServer();
-server.listen(process.env.port || process.env.PORT || 3978, function() {
-    console.log(`\n${ server.name } listening to ${ server.url }.`);
-    console.log(`\nGet Bot Framework Emulator: https://aka.ms/botframework-emulator.`);
-    console.log(`\nTo talk to your bot, open facebook-events.bot file in the Emulator.`);
-});
+const { FacebookBot } = require('./bots/facebookBot');
+// Import required bot services. See https://aka.ms/bot-services to learn more about the different parts of a bot.
+const { BotFrameworkAdapter, MemoryStorage } = require('botbuilder');
```

Comment:
MemoryStorage is not used by this bot.  It can be removed

Reply:
done!